source/kotlin/src/main/kotlin/io/github/cosinekitty/astronomy/Main.kt

```diff
@@ -1,5 +1,10 @@
 package io.github.cosinekitty.astronomy
 
+import java.text.SimpleDateFormat
+import java.util.*
+import kotlin.math.absoluteValue
+import kotlin.math.roundToLong
+
 /*
     Astronomy Engine for Kotlin / JVM.
```

Comment:
Just to note, use of Kotlin outside JVM is also possible but current project structure I brought doesn't support as I don't know [Kotlin Multiplatform](https://kotlinlang.org/docs/multiplatform.html) myself yet which is in its alpha itself.

Reply:
I think that is fine. The main reason I want to add Kotlin support is to support JVM. We can worry about multiplatform support later, when Kotlin Multiplatform is more stable.